src/ApiClient/index.tsx

```diff
@@ -4,17 +4,20 @@
 import { NativeModules } from "react-native";
 import isURL from "validator/es/lib/isURL";
 
-const { APIClient: NativeAPIClient } = NativeModules;
+const { RNNCAPIClient } = NativeModules;
```

Comment:
Are we changing the names from `APIClient` to `RNNCAPIClient` and `GenericClient` to `RNNCGenericClient`? If so - ignore the top comment!